### DISCUSSION TITLE
Fantastic project!

### DISCUSSION THREAD

**Discussion by BartVB** [elsbrock/hetzner-radar#31] [General]
Currently not looking for a new Hetzner server, but have bookmarked this tool for future reference. Looks great! Makes it much easier to find good deals.

Thanks a lot for all the effort that you've put into this!

**Comment by elsbrock** [comment]
Thank you! It still has a few bugs here and there, but happy to hear it's considered useful 🙂

One thing to keep in mind is that prices are displayed without VAT. I should probably add a disclaimer.
